src/hotspot/cpu/aarch64/copy_aarch64.hpp

```diff
@@ -64,28 +64,34 @@ static void pd_zero_to_bytes(void* to, size_t count) {
 "       br      %[t0];\n"                                               \
 "       .align  5;\n"                                                   \
 "0:"                                                                    \
+"       hint    #0x24; // bti j\n"                                               \
```

Comment:
LGTM. Only a few indent issues.

Reply:
@e1iu thanks for review! Updated it in the new commit :)